data/monster/monsters/black_knight.xml

```diff
@@ -7,6 +7,7 @@
 		<flag summonable="0" />
 		<flag attackable="1" />
 		<flag hostile="1" />
+		<flag ignorespawnblock="1" />
```

Comment:
I'm not sure if we should be changing that behaviour by default.

Reply:
What I mean is, there might be projects using this black knight file and having many of them placed on map, so now they'll start respawning wether someone is blocking the spawn or not 🤷

Reply:
perhaps we should just add the functionality and leave the datapack part to user taste

Reply:
I'll add the flag to BK, but with value 0 😎 

💩 
🔽 
🚽 

or just simply remove it, someone will just have to find out what flags are supported 🥇